scripts/quick_start.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+WORKDIR=$(pwd)
+SERVERS=("172.20.0.2" "172.20.0.3" "172.20.0.4" "172.20.0.5")
+CLUSTER_PEERS=(
+    ""
+    "${SERVERS[2]}:2379 ${SERVERS[3]}:2379"
+    "${SERVERS[1]}:2379 ${SERVERS[3]}:2379"
+    "${SERVERS[1]}:2379 ${SERVERS[2]}:2379"
+)
+
+# run xline node by index
+# args:
+#   $1: index of the node
+run_xline() {
+    cmd="/usr/local/bin/xline \
+    --name node${1} \
+    --cluster-peers ${CLUSTER_PEERS[$1]} \
+    --self-ip-port ${SERVERS[$1]}:2379 \
+    --leader-ip-port ${SERVERS[1]}:2379"
+
+    if [ ${1} -eq 1 ]; then
+        cmd="${cmd} --is-leader"
+    fi
+
+    docker exec -e RUST_LOG=debug -d node${1} ${cmd}
+    echo "command is: docker exec -e RUST_LOG=debug -d node${1} ${cmd}"
+}
+
+# run cluster of xline/etcd in container
+# args:
+#   $1: xline/etcd cluster
+run_cluster() {
+    echo cluster starting
+    run_xline 1
+    run_xline 2
+    run_xline 3
```

Comment:
please add & after the above 3 commands, we can start those nodes parallel. if we need to run them one by one, please remove the wait on the next line